src/game/BattleGround/BattleGroundWS.cpp

```diff
@@ -356,6 +356,8 @@ void BattleGroundWS::EventPlayerClickedOnFlag(Player* source, GameObject* target
         }
         else
         {
+	    if(GetFlagState(HORDE) == BG_WS_FLAG_STATE_ON_GROUND)
```

Comment:
missing space between if and bracket

Reply:
Does not having a space there stop that code from running. If so thats really odd because I havent noticed that.

Reply:
codestyle